ascii_art/cli.py

```diff
@@ -15,46 +20,49 @@
               help='Specify the name of the output file instead of using the default. -s is implied.')
 @click.option('-w', '--width', default=100, type=int,
               help='scale the image to fit a custom width')
-@click.option('--credits', is_flag=True, help="Show credits")
-@click.option('--clock', is_flag=True,
-              help='show clock as a colorful animation. resize the terminal or press "q" or "x" to exit the clock.')
 @click.option('--all', is_flag=True, help='convert all supported files')
 @click.option('-c', '--color', type=click.Choice(COLOR_OPTIONS, case_sensitive=False), default='black',
               help='Set output color')
-@click.option('--textinput', is_flag=True, help='Convert Simple Text Into Ascii Text Format, Enter Text After Prompt')
-@click.option('--text', default=None, type=str, help='Convert Simple Text Into Ascii Text Format')
-@click.option('--types', is_flag=True, help='list supported image formats and exit.')
 @click.option('-hr', '--highres', is_flag=True, help='Converts using a wide range of Ascii characters.')
-def main(input_files, reverse, save, output, width, credits, clock, all, color, textinput, text, types, highres):
+def process(input_files, reverse, save, output, width, all, color, highres):
```

Comment:
Please change this to `convert` so that it doesn't override the imported method and so it's descriptive of its function.

Reply:
Oops, sorry about that.

Reply:
I've fixed it.